src/component/Access/access-context.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const AccessContext = React.createContext()
+
+const AccessProvider = ({store, children}) => {
```

Comment:
Could we separate out the AccessProvider into an AccessProvider.tsx file? It would be in line with the naming convention we've gone for with other React components and make it easier to search for. 

I've also been trying to keep all new files in typescript to reduce the workload further down the road. This isn't a must if you feel it will add significant more time to the PR, but nice to keep in the back of the mind when we create new files going forward.